src/en/server-hosting/port-forwarding.md

```diff
@@ -161,4 +161,59 @@ If you did the above steps correctly and your server **still** isn't globally ac
 
 The most common reason for this is that your ISP uses Carrier-grade NAT ("CGNAT") which makes port forwarding impossible. Another possible reason we've heard about is that certain network ports or protocols are automatically blocked by them.
 
-Either way, if this is the case we cannot help you further. You are probably best off contacting your ISP's support or checking their site to see what you can figure out. Some ISPs may allow exemptions to CGNAT, others may do it but will charge you for it, and so on.
+You are probably best off contacting your ISP’s support or checking their site to see what you can figure out. Some ISPs may allow exemptions to CGNAT, others may do it but will charge you for it, and so on.
+
+### So what now?
+
+We can't really do much anymore to help other then suggestions. We made some bellow that we heard have worked.
+
+Essentially from this point on, you will need external help to make your server reachable if you can't get your isp to help.
+
+```admonish info
+We are not the team behind any of these software.
```

Comment:
Reword
"these software" is awkward
"we are not the team behind" -> "we do not endorse or maintain" or some variant thereof